README.md

```diff
@@ -1,21 +1,18 @@
 # proCAT
 
-Project Charging and Analytics Tool
+A Django web app for hosting the Project Charging and Analytics Tool (ProCAT).
 
-This is a Python application that uses [`pip-tools`][pip-tools] for packaging and
-dependency management. It also provides [`pre-commit`][pre-commit] hooks (for
-[`ruff`][ruff] and [`mypy`][mypy]) and automated tests using [`pytest`][pytest] and
-[GitHub Actions]. Pre-commit hooks are automatically kept updated with a dedicated
-GitHub Action, this can be removed and replaced with [pre-commit.ci] if using a public
-repo. The package version is dynamically generated from the most recent git tag using
-[`setuptools-scm`][setuptools-scm].
+This Django project uses:
+
+- [`pip-tools`][pip-tools] for packaging and dependency management.
+- [`pre-commit`](https://pre-commit.com/) for various linting, formatting and static type checking.
+  - Pre-commit hooks are automatically kept updated with [pre-commit.ci](https://pre-commit.ci).
+- [`pytest`](https://pytest.org/) and [GitHub Actions](https://github.com/features/actions).
```

Comment:
The preference is to use the re-usable names for these links that are defined at the bottom of the file. For example:

```suggestion
- [`pytest`][pytest] and [GitHub Actions][GitHub Actions].
```

Reply:
Have added re-usable names for links back in